tslint.json

```diff
@@ -11,6 +11,7 @@
         "indent": [true,
             "spaces"
         ],
+        "inline-comment-spacing": true,

```

Comment:
How about renaming `boolean-trivia` to `literal-argument-comment` and `inline-comment-spacing` to `argument-comment-spacing`?